.eslintrc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "parserOptions": {
+      "ecmaVersion": "latest"
+  },
```

Comment:
what's your opinion about setting a version vs latest? is there any danger with latest?

Reply:
I guess it might be better to set the version just in case there are any breaking changes/unforeseen bugs introduced in a latest version, so I'll do that now